internal/setup/setup.go

```diff
@@ -344,6 +344,13 @@ type yamlRelease struct {
 	V1PubKeys map[string]yamlPubKey `yaml:"v1-public-keys"`
 }
 
+const (
+	// An archive's priority value cannot be larger than MaxArchivePriority.
```

Comment:
I think the comments here are not needed because the variable name is descriptive enough.

Reply:
A'ight. eef309d.